### PR TITLE
Port 2to3 v11

### PR DESCRIPTION
**Pull request by devang-dreambits**
POrting code vai 2to3 and making it compatible for v11.

### CODE REVIEW

File: dbt_shipment_base/models/Shipment.py [hunk 1]
```diff
@@ -4,6 +4,12 @@
 from odoo import models,  fields,  api
 import logging
 
+import sys
+
+PY3 = False
+if sys.version_info[0] > 2:
+    PY3 = True
```
Comment (karan-dreambits) [inline dbt_shipment_base/models/Shipment.py]: Do you find any need to maintain py2.7 code for module specific to odoov11 ?

File: dbt_shipment_base/views/shipment_transporter_view.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <odoo>
-    <data>        
-        
+    <data>
+
         <record id="action_shipment_transporters" model="ir.actions.act_window">
```
Comment (karan-dreambits) [inline dbt_shipment_base/views/shipment_transporter_view.xml]: Why extra space ?

File: dbt_shipment_base/views/res_company_view.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<odoo>
+    <data>
+
```
Comment (karan-dreambits) [inline dbt_shipment_base/views/res_company_view.xml:3]: Why so many extra space ?

**Review by karan-dreambits — CHANGES_REQUESTED** [review]

File: dbt_shipment_base/models/Shipment.py [hunk 4]
```diff
@@ -4,6 +4,12 @@
 from odoo import models,  fields,  api
 import logging
 
+import sys
+
+PY3 = False
```
Comment (karan-dreambits) [inline dbt_shipment_base/models/Shipment.py]: @devang-dreambits  Is this still needed ?

File: dbt_shipment_base/views/res_company_view.xml (hunk 3 above)
Comment (devang-dreambits) [reply dbt_shipment_base/views/res_company_view.xml:3]: spaces are removed.

**Pull request closed without merging**